Source/SPPreferencesUpgrade.m

```diff
@@ -79,7 +79,7 @@ void SPApplyRevisionChanges(void)
 	
 	// This is how you add release notes and run specific migration steps
 	if (recordedVersionNumber < 2061) {
-		[importantUpdateNotes addObject:NSLocalizedString(@"There is a new option in Preferences->Alerts & Logs: \"Show warning before executing a query\". When enabled, you will be prompted to confirm that you want to execte an SQL query or edit a row.", @"Short important release note for new option in Preferences->Alerts & Logs")];
+		[importantUpdateNotes addObject:NSLocalizedString(@"There is a new option in Preferences->Alerts & Logs: \"Show warning before executing a query\". When enabled, you will be prompted to confirm that you want to execute an SQL query or edit a row.", @"Short important release note for new option in Preferences->Alerts & Logs")];
```

Comment:
Shouldn't it be "execute _a_ SQL query"? If we're gonna fix the spelling error we can fix the grammar one too!

Reply:
It depends on if you pronounce it "Sequel" or "S-Q-L".

Reply:
It's "My Ess Que Ell" according to https://dev.mysql.com/doc/refman/8.0/en/what-is-mysql.html.

Reply:
Fascinating, I've always pronounced it "Sequel"! Thanks for the link!